docs/book/v1/async-tasks.md

```diff
@@ -0,0 +1,129 @@
+# Triggering Async Tasks in Swoole HTTP Server
```

Comment:
Since this PR will target a new v2 release, create this under a new `docs/book/v2/` directory.